baseapp/abci.go

```diff
@@ -160,6 +160,24 @@ func (app *BaseApp) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) (res
 	return res
 }
 
+func (app *BaseApp) MidBlock(ctx sdk.Context, height int64) (events []abci.Event) {
+	defer telemetry.MeasureSince(time.Now(), "abci", "mid_block")
```

Comment:
Should we also add tracing to midblock?

Reply:
can you link an example of the tracing? I just included the same telemetry reporting as was present for endblock since midblock is fairly similar

Reply:
tracing rn is only set on sei-chain level i think. To add it for MidBlock you can do it here https://github.com/sei-protocol/sei-chain/blob/master/app/abci.go#L24

Reply:
ah gotcha thanks, i've included the tracing in the sei-chain PR